conf/pipeline/sarek/eva.config

```diff
@@ -0,0 +1,92 @@
+
+// Profile config names for nf-core/configs
+
+params {
+  // Specific nf-core/configs params
+  config_profile_contact = 'James A. Fellows Yates (@jfy133)'
+  config_profile_description = 'nf-core/sarek EVA profile provided by nf-core/configs'
+}
+
+env {
+    _JAVA_OPTIONS = "-XX:ParallelGCThreads=1"
+    OPENBLAS_NUM_THREADS = 1
+    OMP_NUM_THREADS = 1
+}
+
+process {
+  withName:GATK4_APPLYBQSR {
```

Comment:
What about fastqc/snpeff and fgbio?

Reply:
Will update as necessary, FASTQC 'worked' on the test run with no issue strangely...

Reply:
i.e., I've asked the people who want to use the pipeline what tools htey will use from the list :)